### PR TITLE
Fix crash in Audio Client

### PR DESCRIPTION
**Pull request by Atlante45**
Make sure the position and orientation getter lambdas are initialized and won't try to access a null pointer

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 0838d393a0008d4932ed111d2bae88ac68fcd156 available at: https://gist.github.com/35d548cd027d88a4e740<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7495.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7495.dmg

**Comment by Atlante45** [comment]
https://app.asana.com/0/26551373415487/105096043004332

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -598,8 +598,18 @@ Application::Application(int& argc, char** argv, QElapsedTimer& startupTimer) :
     audioThread->setObjectName("Audio Thread");
 
     auto audioIO = DependencyManager::get<AudioClient>();
-    audioIO->setPositionGetter([this]{ return getMyAvatar()->getPositionForAudio(); });
-    audioIO->setOrientationGetter([this]{ return getMyAvatar()->getOrientationForAudio(); });
+    audioIO->setPositionGetter([]{
+        auto audioIO = DependencyManager::get<AvatarManager>();
```
Comment (zzmp) [inline interface/src/Application.cpp]: This should be called `avatarManager`, not `audioIO`. Same in `setOrientationGetter`.

Comment (Atlante45) [inline interface/src/Application.cpp]: Whoops, copy-pastes are evil! Thanks!

**Comment by hifi-gustavo** [comment]
Build console output for commit: 1e389f7c0717c0a5938cbe64e5628b49343d6798 available at: https://gist.github.com/35d548cd027d88a4e740<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7495.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7495.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 238131014bdd800fc87f7186689a455f9e65fae4 available at: https://gist.github.com/35d548cd027d88a4e740<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7495.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7495.dmg

**Comment by ZappoMan** [comment]
smoke test passes

**Pull request merged**